cmd/fontman/commands/install.go

```diff
@@ -2,26 +2,37 @@ package commands
 
 import (
 	"fmt"
+	"fontman/client/pkg/util"
+	"log"
 
 	"github.com/urfave/cli/v2"
 )
 
 // Called if 'install' subcommand is invoked.
-func onInstall(c *cli.Context) error {
+func onInstall(c *cli.Context, style string, ex_style string, global bool) error {
+	err := util.SetupFolders(global)
+
+	if err != nil {
+		log.Fatal(err)
+	}
 	fmt.Println("Installing font(s)...")
 
 	return nil
 }
 
 // Constructs the 'install' subcommand.
 func RegisterInstall() *cli.Command {
+	// TODO: style/ex_style should be arrays of strings; look into how the lib handles multi-parameter argument
 	var style string
+	var ex_style string
```

Comment:
I think this should be `exStyle` or `excludeStyle` just to maintain consistency.